test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/ViewModels/ReadMePreviewViewModelTests.cs

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using NuGet.Common;
+using NuGet.PackageManagement.UI.ViewModels;
+using NuGet.Packaging;
+using NuGet.Test.Utility;
+using Xunit;
+
+namespace NuGet.PackageManagement.UI.Test.ViewModels
+{
+    public class ReadMePreviewViewModelTests
+    {
+        [Fact]
+        public async void ReadMePreviewViewModelTests_NoPackagePathOrPackageId_NoErrorNoReadmeMarkdown()
+        {
+            var readMePreviewViewModel = new ReadMePreviewViewModel();
+
+            await readMePreviewViewModel.LoadReadme("", "");
+
+            Assert.False(readMePreviewViewModel.IsErrorWithReadMe);
+            Assert.True(string.IsNullOrWhiteSpace(readMePreviewViewModel.ReadMeMarkdown));
+            Assert.False(readMePreviewViewModel.CanDetermineReadMeDefined);
+        }
+
+        [LocalOnlyFact]
```

Comment:
I had added it because I was creating the readme locally and the other tests I saw also used localonly. I figured it was because we were creating files locally on the machine. Though I'm no longer doing that so it's gone for now.